src/integration-tests/tests/pocket_ic_integration_test/icrc2_bridge.rs

```diff
@@ -794,3 +795,22 @@ async fn test_icrc2_tokens_roundtrip_with_reschedule_spam() {
     assert_eq!(wrapped_balance, 0);
     assert_eq!(base_balance, ICRC1_INITIAL_BALANCE - ICRC1_TRANSFER_FEE * 3);
 }
+
+#[tokio::test]
+async fn icrc_bridge_stress_test() {
```

Comment:
How long does it take to run these tests? Should we feature gate them?

Reply:
It takes about 90 seconds. Since it is running concurrently with other tests, I think we can keep it running without feature gate.